src/components/Kv/KvLoadingOverlay.vue

```diff
@@ -9,9 +9,11 @@
 <script>
 import _throttle from 'lodash/throttle';
 import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
+import getCacheKey from '@/util/getCacheKey';
 
 export default {
 	name: 'KvLoadingOverlay',
+	serverCacheKey: () => getCacheKey('KvLoadingOverlay'),
```

Comment:
Consider using constants to share strings in these components where they are exact matches, such as `KvLoadingOverlay`, if the name/key match is important here. If the caching is only done based on the `serverCacheKey` and doesn't consider the `name` prop, ignore this comment :).

Reply:
Name is being set here because this component does not contains props.

Reply:
@dyersituations The name/key match isn't important here. They are both used together to create the final cache key: `${component.name}::${component.serverCacheKey(props)}`